### ISSUE TITLE
Tabbing Order E2E Tests - Header Desktop

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4164]
Write e2e tests to check that the tabbing order (pressing the tab key to focus elements) of the header is logical for the desktop version of the app. Create another issue to correct the tabbing order if it is not already.

![image](https://user-images.githubusercontent.com/11735817/63508061-79af7500-c4d9-11e9-9878-29922a7a3ab0.png)

![image](https://user-images.githubusercontent.com/11735817/63508855-158db080-c4db-11e9-93d8-25f3c6fb60ba.png)

_Notes:_
- ) 1. Navigation should start at searchbox. Navigating backwards (ie. shift+tab) should navigate to -1.
- ) 4.5. Navigation in these subcategories should be performed by arrow keys (create another ticket for this...). Pressing tab in these subcategories should go straight to 5.

_Important Note: The category list is currently inaccessible by keyboard._

**Comment by developpeurweb** [comment]
@Zeyber 
Attached below you will find the Initial Keyboard Flow presented by UX Researcher Hana Nagel back in March 2018, based on her research of leading retail websites, the common flow is as presented.
 
The keyboard flow should start on the Group Skippers, then the Logo, which usually vocalizes "xxxx Home" since it will always get you back to the Homepage, and so on, following the sequential order of the elements.
 
Regarding the **subcategories navigation**, the **TAB** key is the standard to move throughout **focusable elements**, you may alternatively use arrow keys as you suggest if that helps you get started,  but it should also work with **TAB** as well to move through the subcategory links. I'll be gathering some examples in [this page](https://wiki.hybris.com/display/spar/Main+Navigation+Specs+for+Desktop) regarding the navigation topic .
 
Please let me know if you have any doubts on this topic before starting to code the navigation behaviour.

![SPA_homepage keyboard nav flow](https://user-images.githubusercontent.com/15113219/63901073-1e95ea80-c9d1-11e9-9427-21e3a4231517.png)
